test/integration/verify/controls/azurerm_webapps.rb

```diff
@@ -0,0 +1,8 @@
+resource_group = attribute('resource_group', default: nil)
```

Comment:
You've already exposed `webapp_name` for the singular resource, so best to make use of it here too.
`webapp_name = attribute('webapp_name', default: nil)`
then
`     its('names')                { should include webapp_name } `